lib/sqlite-store.js

```diff
@@ -1,433 +1,443 @@
+      var ent = args.ent
 
-      var update = true;
-      if( !ent.id ) {
-        update = false;
+      var update = true
+      if (!ent.id) {
+        update = false
         if (ent.id$) {
-          ent.id = ent.id$;
-        } else {
-          ent.id = uuid();
+          ent.id = ent.id$
+        }
+        else {
+          ent.id = Uuid()
         }
       }
 
-      var query;
-      var qlabel = 'update';
+      var query
+      var qlabel = 'update'
 
       if (update) {
         // id received - execute an update
-        query = updatestm(ent);
-      } else {
+        query = updatestm(ent)
+      }
+      else {
         // no id received - execute an insert
-        query = savestm(ent);
-        qlabel = 'insert';
+        query = savestm(ent)
+        qlabel = 'insert'
       }
 
-      self.connection.run(query.text, query.values, function(err, result) {
+      self.connection.run(query.text, query.values, function (err, result) {
         if (err) {
-          return done(err);
-        } else {
-          seneca.log.debug('save/'+qlabel, query, result, desc);
-          return done(null, ent);
+          return done(err)
+        }
+        else {
+          seneca.log.debug('save/' + qlabel, query, result, desc)
+          return done(null, ent)
         }
-      });
+      })
     },
 
-    close: function(args, done){
-      if(self.connection) {
-        self.connection.close(function(err) {
+    close: function (args, done) {
+      if (self.connection) {
+        self.connection.close(function (err) {
           if (err) {
-            return done(err);
-          } else {
-            return done(null, null);
+            return done(err)
           }
-        });
+          else {
+            return done(null, null)
+          }
+        })
       }
 
-      return done(null, null);
+      return done(null, null)
     },
 
-    native: function(args, done){
-      return done(null, self.connecation);
+    native: function (args, done) {
+      return done(null, self.connecation)
     }
-  };
-
-  var selectstm = function(qent,q) {
-    var stm = {};
-    var table = tablename(qent);
-    var params = [];
-    var values = {};
-
-    var w = whereargs(makeentp(qent),q);
-    var wherestr = '';
-
-    if( !_.isEmpty(w) ) {
-      for(var param in w) {
-        var fieldPlaceholder = '$' + param;
-        params.push(param + ' = ' + fieldPlaceholder);
-        values[fieldPlaceholder] = w[param];
+  }
+
+  var selectstm = function (qent, q) {
+    var stm = {}
+    var table = tablename(qent)
+    var params = []
+    var values = {}
+    var w = whereargs(makeentp(qent), q)
+    var wherestr = ''
+
+    if (!_.isEmpty(w)) {
+      for (var param in w) {
+        var fieldPlaceholder = '$' + param
+        params.push(param + ' = ' + fieldPlaceholder)
+        values[fieldPlaceholder] = w[param]
       }
 
-      wherestr = " WHERE " + params.join(' AND ');
+      wherestr = 'WHERE' + params.join('AND')
```

Comment:
this builds a native query string so removing spaces before and after keywords is not ok. you will end up having SELECT*FROMtableWHEREa==bANDc==d